main.css

```diff
@@ -0,0 +1,40 @@
+.main-nav {
+    background-color: #3590F3;
+    color: #F5F5F5;
+    
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    padding-left: 10px;
+}
+
+.nav-links {
+    list-style: none;
+    font-family: "proxima-nova","Helvetica Neue",Helvetica,Arial,sans-serif; /*Figure out a new font*/
+
+    display: flex;
+    justify-content: space-around;
+    flex-grow: 1;
+    max-width: 400px;
+    font-size: 17px;
+}
+
+#logo {
+    color: #F5F5F5;
+    text-decoration: none;
+    font-family: "proxima-nova","Helvetica Neue",Helvetica,Arial,sans-serif; /*Figure out a new font*/
+    font-weight: 600;
+    font-size: 20px;
+    letter-spacing: 2px;
+    font-family: futura-pt;
```

Comment:
Looks like there are 2 different `font-family` declarations here - I'm not sure which one you want to use, but the way it's currently written, the first one will be ignored, so only `futura-pt` will be used.